docker/mysql_tests/scripts/base_tests/journals.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+set -e -x
+
+. /usr/local/export_common.sh
+
+export WALE_S3_PREFIX=s3://mysqlfullxtrabackupbucket
```

Comment:
please use your own bucket =) In theory we can run tests concurrently.

Note1: you should add it in wal-g/docker-compose.yaml before use
Note2: new bucket names uses readable 'mysql_xxx' notation instead of messy 'mysqlxxx'